src/migration/20200907-add_encrypted-columns.js

```diff
@@ -0,0 +1,48 @@
+import { SavedConnection } from "../common/appdb/models/saved_connection";
+import { TableColumn } from 'typeorm';
+
+export default {
+  name: "20200907-add_encrypted-columns",
+  async run(runner) {
+    try {
+      const temp_columns = [
+        new TableColumn({
+          name: 'password_decrypted',
+          type: 'varchar',
+          isNullable: true
+        }),
+        new TableColumn({
+          name: 'sshKeyfilePassword_decrypted',
+          type: 'varchar',
+          isNullable: true
+        }),
+        new TableColumn({
+          name: 'sshPassword_decrypted',
+          type: 'varchar',
+          isNullable: true
+        }),
+      ]
+
+      await runner.startTransaction()
```

Comment:
this might not work (rollback correctly) because sqlite cannot remove columns? Not sure.

Reply:
I think it's best to break this out into two migrations:
1. Create new columns
2. run the update queries (with transaction)